datafortest/datafortest.go

```diff
@@ -0,0 +1,13 @@
+package datafortest
+
+import (
+	"path/filepath"
+	"runtime"
+)
+
+var (
+	_, b, _, _ = runtime.Caller(0)
+
+	// TestDataRootPath is Root path of datafortest package
+	TestDataRootPath = filepath.Dir(b)
```

Comment:
I think you should put test data under the `testdata` directory.  Also the `testdata` directory will be under the package directory. 

First if you use `testdata`, these files are ignored when build.
https://golang.org/cmd/go/#hdr-Test_packages
`The go tool will ignore a directory named "testdata", making it available to hold ancillary data needed by the tests.`

Also if you put `testdata` directory under the package directory, you do not need to get `testdata` path by this complex method.
https://medium.com/@povilasve/go-advanced-tips-tricks-a872503ac859

Reply:
Or how about to make `testdata` directory on root and not to use datafortest.go and get files by relative path (ex ../testdata/file)

Reply:
`datafortest.go` might be noisy a little as you said. I've changed test path to relative, then remove `datafortest.go`. It enables me to change the directory name to `testdata` because it's not go package directory.

Thank you!